README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
```

Comment:
I think `version` would be too generic.  As a user, I'd think "version of what"?  `server_version` feels slightly better because I can probably figure it out from context clues.  I have a personal preference for `postgresql_version` but if it really matters I can change it.